src/FormElement/ScheduleElement.php

```diff
@@ -499,6 +502,16 @@ private function assembleMonthlyOrAnnuallySchedules(): void
         }
 
         $ordinalWrapper->addHtml($enumerations, $selectableDays);
+
+        if ($isAnnually) {
+            $fieldsWrapper->add(HtmlElement::create('div', ['class' => 'note'], [
+                'Use ',
+                new Icon('arrow-left'),
+                ' / ',
+                new Icon('arrow-right'),
+                ' keys to navigate the active element with keyboard.'
```

Comment:
1. This is not translatable
2. ~~navigate the active element with~~ *choose a month by*

Reply:
Seems already adressed by 6d973be.